src/items/associated-items.md

```diff
@@ -98,24 +98,28 @@ Associated functions whose first parameter is named `self` are called *methods*
 and may be invoked using the [method call operator], for example, `x.foo()`, as
 well as the usual function call notation.
 
-If the type of the `self` parameter is specified, it is limited to one of the
-following types:
+If the type of the `self` parameter is specified, it is limited to semantic
```

Comment:
What's a "semantic type"? In the PL research community this has a very specific meaning (which you can read about e.g. in [this paper](https://plv.mpi-sws.org/rustbelt/popl18/paper.pdf)), but I doubt that is what you mean.

Reply:
Indeed, it's not the "semantic type system" notion in PL research. Rather, this means to say that the `Self` here is not meant literally (as in those characters making up `Self`). So it could be a type alias to e.g. `Rc<TheImplementingType>`. Ideas for a better wording?

Reply:
"unfolded type"? "fully expanded type"? "resolved type"?

Reply:
"resolved type" seems good.